ignite/contrib/metrics/cohen_kappa.py

```diff
@@ -0,0 +1,63 @@
+from typing import Callable
+
+import torch
+
+from ignite.metrics import EpochMetric
+
+
+class CohenKappa(EpochMetric):
+    """Compute different types of Cohen's Kappa: Non-Wieghted, Linear, Quadratic.
+    Accumulating predictions and the ground-truth during an epoch and applying
+    `sklearn.metrics.cohen_kappa_score <https://scikit-learn.org/stable/modules/
+    generated/sklearn.metrics.cohen_kappa_score.html>`_ .
+
+    Args:
+        output_transform (callable, optional): a callable that is used to transform the
```

Comment:
Please, remove the type hints
```suggestion
        output_transform: a callable that is used to transform the
```
and same below.